technical/indicators/indicators.py

```diff
@@ -1295,3 +1295,49 @@ def tv_hma(dataframe: DataFrame, length: int = 9, field="close") -> DataFrame:
     dataframe.drop("h", inplace=True, axis=1)
 
     return dataframe
+
+def tv_alma(dataframe: DataFrame, length: int = 8, offset: int = 0, sigma: int = 0, field="close") -> DataFrame:
+    """
+    Source: Tradingview "Arnaud Legoux Moving Average"
```

Comment:
Ideally, we'd want to have a link to the actual script source - as that makes independent comparison simpler.

Searching for this term on tradingview reveals several scripts - assuming they all do things slightly different - it's making it unclear what we should compare against.